File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/attivita/centri_di_costo/Progetto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.attivita.centri_di_costo
{
    public class Progetto : CentroDiCosto
    {
        public string partitaIva { get; set; }
        public string descrizione { get; set; }

        public Progetto()
        {
            classType = costanti.ClassType.Progetto;
            tipo = Tipo.Progetto;
        }
    }
}

File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/attivita/centri_di_costo/Progetto.ts
import {CentroDiCosto, Tipo} from './CentroDiCosto';

export class Progetto extends CentroDiCosto {


    constructor() {
        super();
        this.classType = 'Progetto';
        this.tipo = Tipo.Progetto;
    }
}
